### PR TITLE
Docs:Set execution mode either 'batch' or 'streaming'

### PR DESCRIPTION
**Pull request by Zephyr589**
When we use flink 1.13 or flink 1.14, we should use 'SET execution.runtime-mode = streaming;' and  'SET execution.runtime-mode = batch;'  instead of using 'SET execution.type= streaming ;' and 'SET execution.type= batch;'.

### CODE REVIEW

**Comment by rdblue** [comment]
@openinx, can you review this?

File: docs/flink/flink-getting-started.md [hunk 1]
```diff
@@ -365,10 +365,10 @@ Iceberg support both streaming and batch read in flink now. we could execute the
 
 ```sql
 -- Execute the flink job in streaming mode for current session context
-SET execution.type = streaming
+SET execution.runtime-mode = streaming;
```
Comment (openinx) [inline docs/flink/flink-getting-started.md:368]: Looks good to me, the new sql-client will just use the `runtime-mode` .

**Review by openinx — APPROVED** [review]
LGTM, thanks @Zephyr589 for the contribution.  (Sorry for the delay,  I recently spent a lot of bandwidth in the company's internal system design work).

**Comment by Zephyr589** [comment]
@openinx Never mind ,thanks for your review.

**Pull request merged**